docs/tutorials/tutorial-update-data.md

```diff
@@ -24,33 +24,39 @@ sidebar_label: "Updating existing data"
   -->
 
 
-This tutorial demonstrates how to update existing data, showing both overwrites and appends.
+This tutorial shows you how to update data in a datasource by overwriting existing data and adding new data to the datasource.
 
-For this tutorial, we'll assume you've already downloaded Apache Druid as described in
-the [single-machine quickstart](index.md) and have it running on your local machine.
+## Prerequisites
 
-It will also be helpful to have finished [Tutorial: Loading a file](../tutorials/tutorial-batch.md), [Tutorial: Querying data](../tutorials/tutorial-query.md), and [Tutorial: Rollup](../tutorials/tutorial-rollup.md).
+Before starting this tutorial, download and run Apache Druid on your local machine as described in
+the [single-machine quickstart](index.md).
 
-## Overwrite
+You should also be familiar with the material in the following tutorials:
+* [Tutorial: Loading a file](../tutorials/tutorial-batch.md)
```

Comment:
I think we're getting rid of the "Tutorial: " prefix - at least that's what Charles commented on a review of my latest tutorial.

Reply:
I agree with the change. My preference would be to change this text once the other files get updated so they remain consistent until then.